google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Model.java

```diff
@@ -116,13 +115,13 @@ Builder setTrainingRuns(List<TrainingRun> trainingRunList) {
     }
 
     @Override
-    Builder setLabelColumns(List<StandardSqlField> labelColumnList) {
+    Builder setLabelColumns(List<StandardSQLField> labelColumnList) {
```

Comment:
Where is this annotated with `@BetaApi`? Just want to make sure we don't introduce a breaking change to something that wasn't labeled as such.

Reply:
The feature represents readonly properties provided by the BQ backend, so I marked it as BetaApi on the public getters in ModelInfo.  The internal builder was not annotated this way.

Reply:
Ah, sorry didn't realize this wasn't a public method. SGTM.